src/atomicui/organisms/AuthGeofenceBox/AuthGeofenceBox.tsx

```diff
@@ -126,7 +126,8 @@ const AuthGeofenceBox: React.FC<AuthGeofenceBoxProps> = ({ mapRef, setShowAuthGe
 						exact,
 						sg => setSuggestions(sg),
 						TriggeredByEnum.GEOFENCE_MODULE,
-						AnalyticsEventActionsEnum.AUTOCOMPLETE
+						AnalyticsEventActionsEnum.AUTOCOMPLETE,
+						false
```

Comment:
What is the purpose of this change?

Reply:
removed it, set a default value for that parameter